lib/tasks/replay_submission.rake

```diff
@@ -57,29 +48,3 @@ namespace :replay_submission do
     end
   end
 end
-
-namespace :replay_hmcts_adapter_submission do
```

Comment:
I think we still want this? Unless it's functionally no different from the replay submission job, as it's used to replay submissions to the HMCTS adapter I think

Reply:
I assumed it does the same (maybe in a different way?) to the existing `rake replay_submission:process[<submission_id>]`

The existing task uses the `Delayed::Job` table, and the hmcts one used the `Submission` table. But I don't know the reason behind this.

I can leave the deleted task if we are not sure.

Reply:
I think I'm going to leave the task, based on the background info in this original PR
https://github.com/ministryofjustice/fb-submitter/pull/440